src/renderer/components/widgets/performCommunityAction/PerformCommunityActionComponent.tsx

```diff
@@ -113,8 +114,8 @@ export interface PerformCommunityActionProps {
   handleRedirection: () => void
   initialValue: string
   handleClose: () => void
-  isClosedDisabled?: boolean
   isConnectionReady?: boolean
+  community: Community | null
```

Comment:
Please change the the prop to "isCloseDisabled: boolean" and do `const isCloseDisabled = !community` inside the container as someone will be really confused by why we pass whole community object to the component